Do not hardcode -ldl
XXX? do not link with -lgfortran as this depends on gcc version (is -lgfortran
really needed?)

--- src/CMakeLists.txt~	2011-06-21 20:37:45.000000000 +0200
+++ src/CMakeLists.txt	2011-11-01 14:13:16.000000000 +0100
@@ -166,7 +166,7 @@
 PKG_CONFIG_USE_DEPENDENCY(${LIBRARY_NAME} dynamic-graph-python)
 
 IF(UNIX)
-  TARGET_LINK_LIBRARIES(${LIBRARY_NAME} dl pthread gfortran)
+  TARGET_LINK_LIBRARIES(${LIBRARY_NAME} ${CMAKE_DL_LIBS} pthread)
 ENDIF(UNIX)
 
 INSTALL(TARGETS ${LIBRARY_NAME} 
@@ -204,7 +204,7 @@
 	
 	# Linux dynamic loading library flags
 	IF(UNIX)
-	  TARGET_LINK_LIBRARIES(${LIBRARY_NAME} dl pthread gfortran)
+	  TARGET_LINK_LIBRARIES(${LIBRARY_NAME} ${CMAKE_DL_LIBS} pthread)
 	ENDIF(UNIX)
 
 	PKG_CONFIG_USE_DEPENDENCY(${LIBRARY_NAME} jrl-mal)
